sqlglot/expressions.py

```diff
@@ -5005,7 +5005,7 @@ class Ceil(Func):
 class Coalesce(Func):
     arg_types = {"this": True, "expressions": False}
     is_var_len_args = True
-    _sql_names = ["COALESCE", "IFNULL", "NVL"]
+    _sql_names = ["COALESCE", "IFNULL"]
```

Comment:
Let's not do it like this, but instead pop `"NVL"` off of `Oracle.Parser.FUNCTIONS`

Reply:
Can you please double check redshift's / others' semantics while you're at it?

Reply:
I was only able to find existing test cases for Oracle and Redshift, I think most dialects only support `COALESCE` or IFNULL` but I'll do a pass